lib/elixir/lib/option_parser.ex

```diff
@@ -847,4 +855,16 @@ defmodule OptionParser do
       types[key]
     end
   end
+
+  defp did_you_mean(option, types) do
+    key = String.trim_leading(option, "-")
```

Comment:
I just realized that we also need to convert "-" into "_". Can you please send another PR for that? Thanks!